tests/e2e/volumeproperty_test.go

```diff
@@ -0,0 +1,175 @@
+	valuesAdminCli, err := verification.GetVolumePropertiesAdminCli(volumeName, s.esxIP)
+	c.Assert(err, IsNil, Commentf(valuesAdminCli))
+
+	valuesDockerCli, err := verification.GetVolumePropertiesDockerCli(volumeName, s.dockerHostIP)
+	c.Assert(err, IsNil, Commentf(valuesDockerCli))
+	s.verifyProperties(c, volumeName, valuesAdminCli, valuesDockerCli)
+}
+
+// verifyProperties - verifies admin cli and docker cli values for size, disk format and attached-to-vm field.
+func (s *VolumePropVerificationSuite) verifyProperties(c *C, volumeName, valuesAdmincli, valuesDockerCli string) {
+	log.Printf("Verifying volume properties like size, disk-format and attached-to-vm fields "+
+		"at vm [%s] and esx [%s] for volume - [%s] ", s.dockerHostIP, s.esxIP, volumeName)
+	arrAdminCliValues := strings.Fields(valuesAdmincli)
+	sliceAdminCliValues := arrAdminCliValues[1:len(arrAdminCliValues)]
+	// status := verification.VerifyAttachedStatus(volumeName, s.dockerHostIP, s.esxIP)
+
+	sliceDockerCliValues := s.arrangeDockerCliValues(valuesDockerCli)
+	for i := range sliceDockerCliValues {
+		c.Assert(len(sliceDockerCliValues), Equals, len(sliceAdminCliValues), Commentf("Values mis-match between admin cli and docker cli for volume %s", volumeName))
+		if i == 2 && !s.vmAttached {
+			c.Assert(sliceDockerCliValues[i], Equals, DiskStatusEmpty, Commentf("Value from dockercli is [%s]", sliceDockerCliValues[i]))
+			c.Assert(sliceAdminCliValues[i], Equals, DiskStatusDetached, Commentf("Value from dockercli is [%s]", i, sliceAdminCliValues[i]))
+		} else {
+			c.Assert(sliceDockerCliValues[i], Equals, sliceAdminCliValues[i], Commentf("Value from admincli is [%s] and that from dockercli is [%s]", sliceAdminCliValues[i], sliceDockerCliValues[i]))
+		}
+	}
+	// Get vm name based on ip and compare ith with the docker cli/admin cli only when vm is attached
+	if s.vmAttached {
+		c.Assert(sliceDockerCliValues[len(sliceDockerCliValues)-1], Equals, s.dockerHostName, Commentf("VM name fetched from attached-to-VM field "+
+			"for volume %s is not same as that fetched from its IP.", volumeName))
+	}
+
+}
+
+// arrangeDockerCliValues - returns docker cli values
+// Wrote this method because if volume is not attached, attached-to-vm field
+// return as <no value> for docker cli whereas admin cli returns "detached". So length of array increases
+// for docker cli values and this makes comparison difficult.
+// 'status' indicates if volume is attached/detached
+func (s *VolumePropVerificationSuite) arrangeDockerCliValues(valuesDockerCli string) []string {
+	arrDockerCliValues := strings.Fields(valuesDockerCli)
+	sliceDockerCliValues := arrDockerCliValues[1:len(arrDockerCliValues)]
+	if !s.vmAttached {
+		sliceDockerCliValues[len(sliceDockerCliValues)-2] = sliceDockerCliValues[len(sliceDockerCliValues)-2] + sliceDockerCliValues[len(sliceDockerCliValues)-1]
+		sliceDockerCliValues = sliceDockerCliValues[0 : len(sliceDockerCliValues)-1]
```

Comment:
:-))))), but no Sir, this isn't right at all. Please make those util function return maps and we all can use those to benefit of more tests.